scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -149,252 +196,374 @@ sce_list
+We will end up leveraging these cell type annotations to explore how successful our integration is: After integration, we expect cell types from different libraries to cluster together, rather than being separated by batches. 
 
-Before we proceed, it's critical to remember that integration procedures do _not actually use_ these cell type annotations, and high-confidence annotations are not always (usually!) available for a given dataset.
-If we have annotations, they are a helpful "bonus" for assessing the integration's success, but they are not necessary for performing integration, nor should you expect to always be able to assess integration using them.
+Before we proceed, it's critical to remember that integration methods _do not actually use_ these cell type annotations.
+If we have annotations, they are a helpful "bonus" for assessing the integration's success, but they are not part of the procedure itself.
 
 
 ## Prepare the SCE list for integration
 
-> reviewer note: I need to make a diagram here to show areas in the SCE that need updating, and possibly also a before/after SCE sketch. 
-
 Now that we have a list of processed SCE objects, we need to merge the objects into one overall SCE object which we can provide as input to integration methods.
 A word of caution before we begin: **This merged SCE object is NOT an integrated SCE!** 
+Merging SCEs itself does not perform any batch correction; that's why we proceed to integration next.
 
 To merge SCE objects, we do need to do some wrangling and bookkeeping to ensure compatibility and that we don't lose important information.
-Overall we'll need to take care of these items: 
+Overall we'll want to take care of these items: 
 
-- We should be able to trace library-specific information back to its originating library, including...
-  - Library-specific statistics, e.g. gene-level statistics for a given library found in `rowData`
+1. We should be able to trace library-specific information back to the originating library, including...
   - Cell-level information: Which library is each cell from? 
   Although cells have unique barcode identifiers, these barcodes are only guaranteed to be unique _within_ a library. 
-  Now that we are combining libraries, there's no guarantee we can identify a cell's originating library from its barcode alone.
-- SCE objects should contain the same genes. In other words, each `rowData` table should have the same rownames.
-- SCE columns should match. In other words, the each `colData` should have the same column names.
+  Now that we are merging several libraries, we won't necessarily be able to identify a cell's originating library from its barcode alone.
+  - Library-specific statistics, e.g. gene-level statistics for a given library found in `rowData`
+2. SCE objects should contain the same genes: Each SCE object should have the same row names.
+3. SCE cell metadata columns should match: Each SCE object should have the same column names.
```

Comment:
```suggestion
3. SCE cell metadata columns should match: The column data for each SCE object should have the same column names.
```
Here the column names shouldn't be the same for the columns of the SCE, they should be the same for the colData.